collect_app/src/androidTest/java/org/odk/collect/android/regression/BaseFormTest.java

```diff
@@ -0,0 +1,18 @@
+package org.odk.collect.android.regression;
+
+import android.Manifest;
+
+import androidx.test.rule.ActivityTestRule;
+import androidx.test.rule.GrantPermissionRule;
+
+import org.junit.Rule;
+import org.odk.collect.android.activities.MainMenuActivity;
+
+public class BaseFormTest {
```

Comment:
If this class is used for all regression tests, not only those related just to form filling then we need a more generic name like BaseRegressionTest.

Reply:
and it should be abstract.